loadtest/control/simulcontroller/websocket.go

```diff
@@ -13,51 +16,79 @@ import (
 // This is used to model user behaviour by responding to certain events with
 // the appropriate actions. It differs from userentity.wsEventHandler which is
 // instead used to manage the internal user state.
-func (c *SimulController) wsEventHandler() {
+func (c *SimulController) wsEventHandler(wg *sync.WaitGroup) {
+	semCount := runtime.GOMAXPROCS(0) * 8
```

Comment:
Any reason not to use `runtime.NumCPU` ? We are not setting any custom GOMAXPROCS explicitly .. so should be okay